modules/cluster-wide-proxy-updates.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-wide-proxy-updates_{context}"]
+= Updating your cluster-wide proxy
+
+After configuring your cluster-wide proxy, you can perform additional configurations on the proxy or add an Additional Trust Bundle.
+
+You may need to perform these actions because the cluster-wide proxy was configured after installation, the network address needs to be updated, your Certificate Authorities expired and the Trust Bundle needs to be replaced.
+
+[NOTE]
+====
+The cluster applies the configuration to the cluster’s control plane and worker nodes. This process results in each node in the cluster temporarily being placed into an unschedulable state and drained of its workloads while applying the configuration. Each node will be restarted as part of this process.
+====
+
+.Prerequsites
+* You have the `oc` CLI installed and configured.
+
+.Procedure
+* To create a cluster with a proxy, run the following command:
++
+ifdef::openshift-rosa[]
+[source,terminal]
+----
+$ rosa edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \ <1>
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
+endif::[]
+ifdef::openshift-dedicated[]
+[source,terminal]
+----
+$ rosa edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \ <1>
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
+endif::[]
++
+While the `additional-trust-bundle-file`, `http-proxy`, and `https-proxy` arguments are optional, if you set a `additional-trust-bundle-file` without either an `http-proxy` or `https-proxy` argument, then the additional trust bundle will not be configured to use a proxy.
```

Comment:
I think we can probably leave this sentence out.. or reword it, though I'm not sure how to succinctly describe the effect that solely setting `additional-trust-bundle-file` has. Maybe change it to:

"then the additional trust bundle will still be used for verifying cluster system egress traffic."